contracts/Bank.sol

```diff
@@ -325,4 +337,38 @@ contract Bank is BankStorage, Ownable, Initializable {
         if (_value > 0) return (true, _value, _time);
         return (false, 0, _time);
     }
+
+    /**
+     * @dev Allows admin to add address to keeper role
+     * @param keeper address of new keeper
+     */
+    function addKeeper(address keeper) external {
+        require(keeper != address(0), "operation not allowed");
+        grantRole(KEEPER_ROLE, keeper);
+    }
```

Comment:
Is a require or modify needed here? Is this only ADMIN_ROLE?

Reply:
is not needed, implementation take care of checking. Added a test for completion on this question